itest/lnd_network_test.go

```diff
@@ -22,9 +22,9 @@ func testNetworkConnectionTimeout(ht *lntest.HarnessTest) {
 		testPub = "0332bda7da70fefe4b6ab92f53b3c4f4ee7999" +
 			"f312284a8e89c8670bb3f67dbee2"
 
-		// testHost is a non-routable IP address. It's used to cause a
-		// connection timeout.
-		testHost = "10.255.255.255"
```

Comment:
was this some random ip or is it well-known in networking that this exact address is unreachable?

Reply:
This is a well-known reserved address, see [private network](https://en.wikipedia.org/wiki/Private_network).